### ISSUE TITLE
Geometry_Engine: NurbSurface not created properly (Knots are not defined)

### ISSUE THREAD

**Issue by adecler** [BHoM/BHoM_Engine#359]
### This issue has been archived 

Historic issue archive can be found [here](https://burohappold.sharepoint.com/sites/BHoM/01_Master%20File/Forms/AllItems.aspx?id=%2Fsites%2FBHoM%2F01_Master%20File%2F999_BHoM_GitHub_History_Archive)

**Comment by pawelbaran** [comment]
@IsakNaslundBh @epignatelli can we close this one as superseded??

**Comment by IsakNaslundBh** [comment]
think this is still an issue though? very low on the priority, but we can not create nurbs surfaces. Should probably be changed to featur_minor though

**Comment by al-fisher** [comment]
The create methods referred to here in this legacy issue were removed at the end of 2018 as part of the change of strategy around NURBS. 

We can represent NURBS Surfaces in BHoM, (see [BH.oM.Geometry.NurbsSurface](https://github.com/BHoM/BHoM/blob/develop/Geometry_oM/Surface/NurbsSurface.cs)) 
And convert to and from proprietary representations, but we removed these methods for complex creation and manipulation native in BHoM in 2018 - recommending using Rhino/Grasshopper to perform those geometrical operations to then convert to BHoM for interop and exchange.

See commit: `Nurbs Create methods switched to NotImplemented` 
https://github.com/BHoM/BHoM_Engine/commit/46a93208d14ea7ec4c40b69c51ef466d4aca72bf#diff-ef74a1593a07a0e609d2d82277a2ed5a380425e75bdd51e9c4efea4f941f248a

Closing issue as now moot. 
As per @IsakNaslundBh's [comment above](https://github.com/BHoM/BHoM_Engine/issues/359#issuecomment-470616135) new feature issue could be raised in future, but not priority based on current strategy as outlined above.
